packages/dds/tree/src/feature-libraries/typed-schema/typedTreeSchema.ts

```diff
@@ -224,7 +224,10 @@ export type FlexMapFieldSchema = FlexFieldSchema<
  *
  * @sealed
  */
-export class FlexFieldSchema<out TKind extends FlexFieldKind = FlexFieldKind> {
+export class FlexFieldSchema<
```

Comment:
Do not add any strong schema aware typing to FlexTree schema. I've just finished removing nearly all such typing as work leading up to removing of flex tree schema entirely.

Also, avoid adding data to flex-tree schema (that not  is not available in the stored schema). Anything you add to it will making removing flex-tree schema harder.